config/prow/cluster/monitoring/BUILD.bazel

```diff
@@ -19,6 +19,7 @@ release(
     component("alertmanager_rbac", MULTI_KIND),
     component("alertmanager_expose", MULTI_KIND),
     component("prow", "alertmanager"),
+    component("alertmanager-prow", "config"),
```

Comment:
I think this assumes/requires the contents of `alertmanager-prow_config.yaml` to only be `kind: Config`. I'm not certain that this would cause problems, but either way it would probably be better to specify `alertmanagerconfig` here and change the file name to match.

Reply:
good point, done